fastcrypto/src/bls12377.rs

```diff
@@ -150,8 +150,8 @@ pub struct BLS12377PrivateKey {
 #[derive(Debug, Serialize, Deserialize)]
 #[serde(tag = "type")] // necessary so as not to deser under a != type
 pub struct BLS12377KeyPair {
-    name: BLS12377PublicKey,
-    secret: BLS12377PrivateKey,
+    pub name: BLS12377PublicKey,
+    pub secret: BLS12377PrivateKey,
```

Comment:
Are we sure about seeing the secret `pub`?

Reply:
I've refactored such that the mksr implementation for bls12377 lives inside the module, so this is not needed anymore.